src/views/CoCurricularTranscript/guestTranscript.tsx

```diff
@@ -0,0 +1,116 @@
+import { MembershipHistory, MembershipHistorySession } from 'services/user';
+import { StudentProfileInfo } from 'services/user';
+import { Class } from 'services/peopleSearch';
+import { Participation } from 'services/membership';
+import { TranscriptItems } from 'services/transcript';
+
+const participation1: Participation = Participation.Member;
```

Comment:
I don't think this needs to be an intermediate variable. You could just replace the one spot that it's referenced with the `Participation.Member` value directly.